tokio-trace-core/src/callsite.rs

```diff
@@ -43,9 +48,16 @@ pub trait Callsite: Sync {
     fn metadata(&self) -> &Meta;
 }
 
+/// Uniquely identifies a [`Callsite`](::callsite::Callsite).
+///
+/// Two `Identifier`s are equal if they both refer to the same callsite.
+#[derive(Clone)]
+pub struct Identifier(&'static Callsite);
```

Comment:
I'm open to 🚲🏚 on the name of this type. `Id` was also considered (given that `span::Id` will be renamed to just `Span` soon (#126)).

Reply:
`Identifier` is good 👍